Doc/tools/templates/dummy.html

```diff
@@ -9,6 +9,16 @@
 
 In extensions/c_annotations.py:
 
+{% trans %} Part of the {% endtrans %}
```

Comment:
@AA-Turner sphinx-lint complains about the trailing whitespaces here and [here](https://github.com/python/cpython/pull/107680/files#diff-58c6319afa976708e1745536a41fd0c50fa2118cfcbb56275ab08dbaf5bbe061R19), and if I remove these whitespace in the translation, the HTML ends up like this: (no spaces in-between)

![image](https://github.com/python/cpython/assets/75242929/4fe53d7e-dfa5-4450-88b8-a95ffafd7c52)

![image](https://github.com/python/cpython/assets/75242929/183c76f1-9825-4ff0-80bb-1f6ac766ceac)

![image](https://github.com/python/cpython/assets/75242929/97c170ca-691f-496c-8e05-c177bae62a14)

and so on

Could we remove these trailing whitespaces here, and insert them at the beginning of [``Limited API``](https://github.com/python/cpython/pull/107680/files#diff-58c6319afa976708e1745536a41fd0c50fa2118cfcbb56275ab08dbaf5bbe061R13) and [``Stable ABI``](https://github.com/python/cpython/pull/107680/files#diff-58c6319afa976708e1745536a41fd0c50fa2118cfcbb56275ab08dbaf5bbe061R14)?